pino.d.ts

```diff
@@ -200,6 +201,32 @@ declare namespace P {
         dest?: string | number | DestinationObjectOptions | DestinationStream | NodeJS.WritableStream,
     ): SonicBoom;
 
+    interface TransportTargetOptions<TransportOptions = Record<string, any>> {
+        target: string
+        options: TransportOptions
+        level: LevelWithSilent
+    }
+
+    interface TransportBaseOptions<TransportOptions = Record<string, any>> {
+        options?: TransportOptions
+        worker?: WorkerOptions & { autoEnd?: boolean}
+    }
+
+    interface TransportSingleOptions<TransportOptions = Record<string, any>> extends TransportBaseOptions<TransportOptions>{
+        target: string
+    }
+
+    interface TransportMultiOptions<TransportOptions = Record<string, any>> extends TransportBaseOptions<TransportOptions>{
+        targets: readonly TransportTargetOptions<TransportOptions>[]
+    }
+
+    // ToDo https://github.com/pinojs/thread-stream/issues/24
+    type ThreadStream = any
```

Comment:
Considering that ThreadStream is at a pre-release version and might potentially change a lot, I don't think it makes much sense to in-line its type definitions in pino, especially since it is likely to be simply fed into pino and not manipulated directly.
I'll work on https://github.com/pinojs/thread-stream/issues/24 when there is green light to do so, and replace this type with a reference to bundled type, but I don't think it should be a blocker here.